src/main/scala/nl/knaw/dans/easy/multideposit/DDM.scala

```diff
@@ -21,24 +21,22 @@ object DDM {
   val profileFields: Dictionary =
     Map("DC_TITLE" -> "dc:title",
       "DC_DESCRIPTION" -> "dcterms:description",
-      "DC_CREATOR" -> "dc:creator",
+      "DC_CREATOR" -> "dc:creator", // TODO verwijder
```

Comment:
`DC_CREATOR` and `DC_CONTRIBUTOR` are referenced in `multi-deposit-instructions.md` as deprecated. In order to remember their removal in the (near?) future, I added these TODOs here.